src/utils.ts

```diff
@@ -8,12 +8,14 @@ export const getDescriptions = (editor: TextEditor) => {
 	const currentLine = editor.selection.active.line + 1
 	const relativeWithLine = `${relativePath}:${currentLine}`
 	const fileName = relativePath.replace(/^.*[\\\/]/, '')
+  const absolutePathWithLine = `file://${absolutePath}#${currentLine}`
 
```

Comment:
```suggestion
	const absolutePathWithLine = `${editor.document.uri.toString()}#${currentLine}`
```

should do the same, and looks less hacky.

Reply:
let's do this and we can merge the PR after package-lock rollback

Reply:
done